configuration/viperConfig.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func LoadConfiguration() error {
```

Comment:
Please review the error handling in this method. What happens if Viper is unable to complete reading the file or encounters an unsupported character? It seems we are currently only handling the `configFileNotFoundError`.

You can keep it as simple as `if err != nil {return err}` letting the caller decide what to do.

Not being able to load the configuration would be a candidate for a log.Fatal or panic.